### DISCUSSION TITLE
problem with exec command output

### DISCUSSION THREAD

**Discussion by sannaroby** [testcontainers/testcontainers-node#704] [Q&A]
I'm using testcontainers-node (v10.4.0) to start a container from the "localstack-pro" (v3.0.0) image, because I need to test the integration with some aws services.

```
let localStackContainer = await new LocalstackContainer("localstack/localstack-pro:3.0.0")
      .withEnvironment({ SERVICES: "timestream-write,timestream-query,sqs" })
      .withEnvironment({ DEBUG: 1 })
      .withEnvironment({ LOCALSTACK_API_KEY: process.env.LOCALSTACK_API_KEY })
      .withStartupTimeout(DEFAULT_TIMEOUT_MILLISECONDS)
    .start();
```

Then I run some commands in this container: for example, in the code I need to create one sqs queue, so I run a command in this way:

`let response = await localStackContainer.exec(["sh", "-c", "awslocal sqs create-queue --queue-name test-queue"]);`

The received output (response.output property in the example above) is not a valid json and I can't parse it:

```{"output":"\u0002\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0001\u0000\u0000\u0000\u0000\u0000\u0000K{\"QueueUrl\": \"http://sqs.us-east-1.localhost:4566/000000000000/test-queue\"}","exitCode":0}```

Have you idea about I can solve this problem?

Thanks for your support.

**Comment by cristianrgreco** [comment]
Hi @sannaroby 

I've tried to reproduce this (using the non-pro version of the image), and the output is as expected:

![image](https://github.com/testcontainers/testcontainers-node/assets/10348798/c61b60ba-e233-4cc1-a8e3-90eb78a0219d)

Could you give more info, e.g node version, testcontainers version, container runtime, version, etc.

**Reply by sannaroby** [reply]
Ni @cristianrgreco, thanks for your check.
I'm using: 
- node v14.21.3 
- npm v6.14.18
- testcontainers/localstack v10.5.0

Anyway, I've done further tests and I've the problems only during the jenkins build where my team use "[kubedock](https://github.com/joyrex2001/kubedock)" to orchestrate the containers used in the integration-test on the ci kubernetes cluster. I suppose that, somehow, the api of the kubedock component "dirty" the json returned by the exec command
I think I'll have to do some testing with this setup.

Thanks again for the support, I don't want to waste your time further, feel free to close/delete this request of mine.
